README.txt

```diff
@@ -24,6 +24,11 @@ Gallerya transforms the WordPress native post gallery into a full fledged slides
 
 == Changelog ==
 
+= 2.2.8 =
+2020-03-30
+
+* Added caching for product image/variation sliders on product listings.
```

Comment:
This is not caching anything anymore, as it we first intended, it just update the queries for performance improvement. Maybe we can say something like: "Updated queries for performance improvement for product image/variation sliders on product listings."